CMakeLists.txt.in

```diff
@@ -7,7 +7,7 @@ project(verilogAST-download NONE)
 include(ExternalProject)
 ExternalProject_Add(verilogAST
   GIT_REPOSITORY    https://github.com/leonardt/verilogAST-cpp.git
-  GIT_TAG           master
+  GIT_TAG           inline-slice-id
```

Comment:
Did you mean to leave this in?

Reply:
This is just needed for the build to pass until the upstream verilogAST branch has been merged to master